docs/Reference.md

```diff
@@ -45,6 +45,10 @@ All HTTP-layer services and protocols (e.g. http, websocket) within the Discord
 
 Discord utilizes Twitter's [snowflake](https://github.com/twitter/snowflake/tree/snowflake-2010) format for uniquely identifiable descriptors (IDs). These IDs are up to 64bits in size (e.g. a uint64) and therefore are always returned as strings in the API to prevent integer overflows in some languages. Snowflake IDs are guaranteed to be unique across all of Discord, except in some unique scenarios in which child objects share their parents ID.
 
+## Nullable Resource Fields
+
+Resource fields that may be null have types that are prefixed with a question mark.
```

Comment:
optional parameters are sometimes documented with a postfixed question mark as well on the field name.